v3/integrations/nrfasthttp/example/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/newrelic/go-agent/v3/integrations/nrfasthttp"
+
+	newrelic "github.com/newrelic/go-agent/v3/newrelic"
+
+	"github.com/valyala/fasthttp"
+)
+
+func main() {
```

Comment:
I would like to see a client example too. They are different products, and we should show customers how to use both.